src/xdocs/writingchecks.xml

```diff
@@ -663,9 +663,8 @@ public class MethodLimitCheck extends AbstractCheck
         There are basically only a few limits for Checkstyle:
       </p>
       <ul>
-        <li>Java code should be written with
-          <a href="https://en.wikipedia.org/wiki/ASCII">ASCII</a> characters only, no UTF-8
-          support.
+        <li>Java identifiers should be written with
```

Comment:
Why the change from `code` to `identifiers`?

An identifier is things the user name like classes and variables.

I assume "code" expands to keywords. It is possible to write these keywords with unicode which we don't support.

The following example compiles on Eclipse:
````
public cl\u0061ss Test {}
````
but can't pass Checkstyle.

I don't see this change being correct.

Reply:
See @romani 's message at https://github.com/checkstyle/checkstyle/pull/8565#issuecomment-723456747

Reply:
fixed.